test/FileService.Data.UnitTests/AddNewFileRepositoryTests.cs

```diff
@@ -9,10 +9,10 @@ namespace LT.DigitalOffice.FileService.Data.UnitTests
 {
     public class AddNewFileRepositoryTests
     {
```

Comment:
I think will be better AddFileRepositoryTests without "New"

Reply:
Fixed